### PR TITLE
assign control function

### PR DESCRIPTION
**Pull request by MarcHafner**

### CODE REVIEW

**Comment by MarcHafner** [comment]
Plot function added to the gr50 module;
GR_AUC and EC50 corrected to GR_AOC and GEC50

File: SRC/python/gr50/__init__.py [hunk 1]
```diff
@@ -267,20 +320,20 @@ def gr_metrics(data, alpha=0.05):
     ...             ['B', 1.0, 1.04], ['B', 10.0, 0.936]],
     ...            columns=['drug', 'concentration', 'GRvalue'])
     >>> print gr_metrics(data)
-      drug      GR50   GRmax    GR_AUC      EC50     GRinf      Hill  \\
-    0    A  0.114025  0.0188  0.481125  0.110411  0.018109  1.145262   
-    1    B       inf  0.9360  0.026375  0.000000  0.971000  0.010000   
+      drug      GR50   GRmax    GR_AOC     GEC50     GRinf      h_GR   \\
```
Comment (jmuhlich) [inline SRC/python/gr50/__init__.py:323]: The spaces (including trailing spaces) must match exactly or the doctest check will fail. Note that an extra space was introduced between `h_GR` and the `\\`, and the three trailing spaces on the next two lines were lost.

Note that you can run `python -m doctest gr50/__init__.py` to run the checker to see if you've got it right.

File: SRC/python/gr50/__init__.py [hunk 2]
```diff
@@ -267,20 +320,20 @@ def gr_metrics(data, alpha=0.05):
...
+    0    A  0.114025  0.0188  0.481125  0.110411  0.018109  1.145262
+    1    B       inf  0.9360  0.026375  0.000000  0.971000  0.010000
     <BLANKLINE>
-                 r2      pval
```
Comment (jmuhlich) [inline SRC/python/gr50/__init__.py:274]: Also you lost the two trailing spaces on these three lines.

**Review by jmuhlich — APPROVED** [review]
I noted some problems with the doctest blocks, but those can be resolved later.

**Pull request merged**